Src/Base/AMReX.H

```diff
@@ -89,6 +89,10 @@ namespace amrex
     void ExecOnFinalize (PTR_TO_VOID_FUNC);
     void ExecOnInitialize (PTR_TO_VOID_FUNC);
 
+    //! This shuts up the compiler about unused variables
+    template <class... Ts> AMREX_GPU_HOST_DEVICE
```

Comment:
I usually also put a force inline on that one.

Reply:
Sounds good.  Will update.

Reply:
Looks like we have take `constexpr`, because conexpr void is not supported in C++11.

Reply:
Yes right, in C++11 mode I used a const fallback originally: https://github.com/alpaka-group/alpaka/pull/582